Documentation/gettingstarted/policy-creation.rst

```diff
@@ -47,32 +48,42 @@ To observe policy audit messages, follow these steps:
          If you installed Cilium via ``helm install``, then you can use ``helm
          upgrade`` to enable Policy Audit Mode:
 
-         .. parsed-literal::
+         .. code-block:: shell-session
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace $CILIUM_NAMESPACE \\
+                --reuse-values \\
+                --set policyAuditMode=true
 
-            helm upgrade cilium |CHART_RELEASE| \\
-               --namespace $CILIUM_NAMESPACE \\
-               --reuse-values \\
-               --set policyAuditMode=true
 
-#. Apply a default-deny policy:
+Enable Policy Audit Mode (specific endpoint)
+============================================
 
-   .. literalinclude:: ../../examples/minikube/sw_deny_policy.yaml
+Cilium can enable Policy Audit Mode for a specific endpoint. This may be helpful when enabling
+Policy Audit Mode for the entire daemon is too broad. Enabling per endpoint will ensure other
+endpoints managed by the same daemon are not impacted.
 
-   CiliumNetworkPolicies match on pod labels using an "endpointSelector" to identify the sources and destinations to which the policy applies.
-   The above policy denies traffic sent to any pods with label (``org=empire``).
-   Due to the Policy Audit Mode enabled above, the traffic will not actually
-   be denied but will instead trigger policy verdict notifications.
+This approach is meant to be temporary.  **Restarts of the Cilium pod will reset the Policy Audit
+Mode to match the daemon's configuration.**
 
-   To apply this policy, run:
+Policy Audit Mode is enabled for a given endpoint by modifying the endpoint configuration via
+the ``cilium`` tool on the endpoint's Kubernetes node. The steps include:
 
-   .. parsed-literal::
+#. Determine the endpoint id on which Policy Audit Mode will be enabled.
+#. Identify the Cilium pod running on the same Kubernetes node corresponding to the endpoint.
+#. Using the Cilium pod above, modify the endpoint config by setting ``PolicyAuditMode=Enabled``.
 
-       $ kubectl create -f \ |SCM_WEB|\/examples/minikube/sw_deny_policy.yaml
-       ciliumnetworkpolicy.cilium.io/empire-default-deny created
+The following shell commands performs these steps:
+
+.. code-block:: shell-session
+
+   $ export PODNAME=deathstar
+   $ export NODENAME=$(kubectl get pod -o json | jq -r --arg pod $PODNAME '.items | .[] | select(.metadata.name==$pod) | .spec.nodeName')
+   $ export ENDPOINT=$(kubectl get cep -o json | jq -r --arg pod $PODNAME '.items | .[] | select(.metadata.name==$pod) | .status.id')
```

Comment:
Double quotes around `$PODNAME` would be better, and I think the `jq` commands can be simplified as follows (but I' have NOT double-checked):

```suggestion
   $ export NODENAME=$(kubectl get pod -o json | jq -r --arg pod "$PODNAME" '.items[] | select(.metadata.name==$pod).spec.nodeName')
   $ export ENDPOINT=$(kubectl get cep -o json | jq -r --arg pod "$PODNAME" '.items[] | select(.metadata.name==$pod).status.id')
```

Same for the section on disabling policy audit mode.

Reply:
quoted and simplified